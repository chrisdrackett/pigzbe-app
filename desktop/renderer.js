import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
