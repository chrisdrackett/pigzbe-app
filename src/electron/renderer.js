import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
