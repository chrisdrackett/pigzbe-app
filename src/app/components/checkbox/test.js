import React from 'react';
import renderer from 'react-test-renderer';
import Checkbox from './';

const props = {
    value: false,
    onValueChange: () => {}
};

describe('Checkbox', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Checkbox {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
