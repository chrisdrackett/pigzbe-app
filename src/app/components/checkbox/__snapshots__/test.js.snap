// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders correctly 1`] = `
<View
  onClick={[Function]}
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "borderColor": "rgb(255, 255, 255)",
        "borderRadius": 15,
        "borderStyle": "solid",
        "borderWidth": 1,
        "height": 30,
        "justifyContent": "center",
        "width": 30,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgb(255, 255, 255)",
          "borderRadius": 8,
          "height": 16,
          "opacity": 0.3,
          "width": 16,
        }
      }
    />
  </View>
</View>
`;
